core/templates/pages/topics-and-skills-dashboard-page/templates/delete-topic-modal.template.html

```diff
@@ -1,19 +1,28 @@
-<div class="modal-header">
-  <h3>
-    Delete Topic
-  </h3>
-</div>
-<div class="modal-body" style="color: #666666">
-  Are you sure you want to delete <strong style="color: black"><[topicName]></strong>? This cannot be recovered.
-</div>
-<div class="modal-footer">
-  <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-danger protractor-test-confirm-topic-deletion-button" ng-click="confirm()">
-    <span>Delete Topic</span>
-  </button>
+<div class="oppia-delete-topic-container">
+  <div class="modal-header">
+    <h3>
+      Delete Topic
+    </h3>
+  </div>
+  <div class="modal-body oppia-delete-topic-text-container">
+    Are you sure you want to delete <strong class="oppia-topic-name-text"><[topicName]></strong>? This cannot be recovered.
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
+    <button class="btn btn-danger protractor-test-confirm-topic-deletion-button" ng-click="confirm()">
+      <span>Delete Topic</span>
+    </button>
+  </div>
 </div>
 
 <style>
+  .oppia-delete-topic-container .oppia-delete-topic-text-container {
+    color: #666666;
```

Comment:
Use #666, we use only three values if the values are repetetive since they mean the same.

Reply:
Done.